pkg/constant/labels.go

```diff
@@ -80,27 +80,18 @@ func GetComponentDefLabel(compDefName string) map[string]string {
 	}
 }
 
-// GetClusterCompDefLabel returns the label for ClusterComponentDefinition (refer clusterDefinition.Spec.ComponentDefs[*].Name)
-// TODO:ClusterCompDef will be deprecated in the future
-func GetClusterCompDefLabel(clusterCompDefName string) map[string]string {
-	return map[string]string{
-		AppComponentLabelKey: clusterCompDefName,
-	}
-}
-
-// GetWorkloadTypeLabel returns the label for WorkloadType (refer clusterDefinition.Spec.ComponentDefs[*].WorkloadType)
-// TODO:workloadType will be deprecated in the future
-func GetWorkloadTypeLabel(workloadType string) map[string]string {
+// GetShardTemplateNameLabel returns the shard template name label for component generated from shardSpec
+func GetShardTemplateNameLabel(shardTplName string) map[string]string {
```

Comment:
there are many variables or labels named shard template, but they should be actually named sharding

Reply:
> there are many variables or labels named shard template, but they should be actually named sharding

rename done